code/modules/unit_tests/_unit_tests.dm

```diff
@@ -55,8 +55,10 @@
 #define TEST_DEFAULT 1
 /// After most test steps, used for tests that run long so shorter issues can be noticed faster
 #define TEST_LONGER 10
-/// This must be the last test to run due to the inherent nature of the test iterating every single tangible atom in the game and qdeleting all of them (while taking long sleeps to make sure the garbage collector fires properly) taking a large amount of time.
-#define TEST_CREATE_AND_DESTROY INFINITY
+/// This must be the one of last tests to run due to the inherent nature of the test iterating every single tangible atom in the game and qdeleting all of them (while taking long sleeps to make sure the garbage collector fires properly) taking a large amount of time.
+#define TEST_CREATE_AND_DESTROY 9001
+/// For tests that rely on create and destroy having iterated through every (tangible) atom so they don't have to do something similar.
+#define TEST_AFTER_CREATE_AND_DESTROY INFINITY
```

Comment:
Please document here, create and destroy will absolutely break the test platform, anything that relies on its shape cannot come after it

Reply:
Oh, good catch. I didn't think about that!